riscv/Cargo.toml

```diff
@@ -56,6 +57,7 @@ serde = { version = "1.0", default-features = false, features = [
   "derive",
   "rc",
 ] }
+serde_cbor = "0.11.2"
```

Comment:
note to self: can probably be removed

Reply:
why not remove?

Reply:
will keep cbor for now because of cross platform